tests/integration.rs

```diff
@@ -8116,6 +8117,7 @@ mod windows {
     }
 
     #[test]
+    #[ignore = "See #5532"]
```

Comment:
This bug number is about migration  doesn't seem appropriate for this test.

Reply:
Bo created an issue for this https://github.com/cloud-hypervisor/cloud-hypervisor/issues/5878 . But most of the details are in this PR. which one you suggest?

Reply:
Right, these are windows related tests, what would be the better way to mention there?

Reply:
Please create an issue to track these disabled tests on Windows worker, and refer the new issue here.

Reply:
Sure